assets/sass/components/dashboard/_googlesitekit-DashboardPageSpeed.scss

```diff
@@ -161,4 +161,45 @@
 			}
 		}
 	}
+
+	.googlesitekit-pagespeed--recommendations {
+		box-shadow: inset 0 1px 0 rgba(0, 0, 0, 0.1);
+		padding: $grid-gap-phone $grid-gap-phone 0;
+
+		.googlesitekit-pagespeed-recommendations__title {
+			color: $c-black;
+			font-size: 1rem;
+			letter-spacing: 0.1px;
+			line-height: 1.5rem;
+		}
+
+		.googlesitekit-accordion {
+			border-top-left-radius: 8px;
+			border-top-right-radius: 8px;
+
+			& + .googlesitekit-accordion {
+				border-radius: 0;
+			}
+
+			&:last-of-type {
+				border-bottom-left-radius: 8px;
+				border-bottom-right-radius: 8px;
+			}
+		}
+
+		.googlesitekit-pagespeed-recommendations__title,
+		.googlesitekit-accordion:last-child {
+			margin-bottom: $grid-gap-phone;
```

Comment:
This margin only "shows up" for the refactored widget, but not for the current production version (not using the Widget API). We need to tweak some CSS to get the same appearance for the "old" (current) version.
![Screenshot 2020-12-01 at 17 24 28](https://user-images.githubusercontent.com/3531426/100816024-16a9cf00-33fa-11eb-96ce-fa54cba4acf9.png)

Reply:
Yep, fixed.